Dockerfile-alpine.template

```diff
@@ -1,29 +1,48 @@
 FROM alpine:0.0
 
+ARG CHECKSUM=
+
 ENV NODE_VERSION 0.0.0
 
 RUN addgroup -g 1000 node \
     && adduser -u 1000 -G node -s /bin/sh -D node \
     && apk add --no-cache \
         libstdc++ \
     && apk add --no-cache --virtual .build-deps \
-        binutils-gold \
-        curl \
-        g++ \
-        gcc \
-        gnupg \
-        libgcc \
-        linux-headers \
-        make \
-        python \
-  # gpg keys listed at https://github.com/nodejs/node#release-keys
-  && for key in \
-    "${NODE_KEYS[@]}"
-  ; do \
-    gpg --batch --keyserver hkp://p80.pool.sks-keyservers.net:80 --recv-keys "$key" || \
-    gpg --batch --keyserver hkp://ipv4.pool.sks-keyservers.net --recv-keys "$key" || \
-    gpg --batch --keyserver hkp://pgp.mit.edu:80 --recv-keys "$key" ; \
-  done \
+      curl \
+    && ARCH= && alpineArch="$(arch)" \
+      && case "${alpineArch##*-}" in \
```

Comment:
This case statement should also be setting `CHECKSUM` to the correct value for that architecture (so no need for the `ARG`). Otherwise it won't work on a non-x64 build and we'll have to revisit this section once there are non-amd64 builds on `unofficial-builds.nodejs.org`.

We have a similar block for golang; you can see the code to generate it [here](https://github.com/docker-library/golang/blob/2f6469ffe955721dd25e4cbb3013506659998aad/update.sh#L69-L82).

Reply:
Done!

Reply:
Done!